src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -367,7 +367,7 @@ const std::vector<impl_desc_type>& Convolution::getDefaultImplPriority() {
     return priorities;
 }
 
-const bool Convolution::isBrgConvAvailable = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core);
+const bool Convolution::isBrgConvAvailable = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx2);
```

Comment:
I guess that condition should be restored back. As well as other changes inside conv.cpp

Reply:
Yes. Will updated.